openstack/elektra/values.yaml

```diff
@@ -19,7 +19,7 @@ monsoon_dashboard_mail_user: defined_in_secrets
 monsoon_dashboard_mail_password: defined_in_secrets
 
 monsoon_dashboard_avatar_url: https://avatars.wdf.sap.corp/avatar/#{current_user.name}?size=24x24
-monsoon_dashboard_cam_url: https://spc.ondemand.com/sap/bc/webdynpro/a1sspc/cam_wd_central
+monsoon_dashboard_cam_url: https://cam.int.sap/cam/ui/admin?#
```

Comment:
Hey, is `?#`really needed at the end?

Reply:
this the link that was provided and it worked but you are right I guess we do not need that